### PR TITLE
Reversed words of a string Question list

### PR DESCRIPTION
**Pull request by SBrandon0432**
image

### CODE REVIEW

File: src/Components/DisplayReveresQuestions/utils.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+export const sortReverseOrderAndReverseString = ({ questions }: Props) => {
+    let reversedOrder = questions.sort((a, b) => {
+        if (a.question > b.question) return -1;
+        if (a.question > b.question) return 1;
+        return 0;
+    });
+
+    reversedOrder.forEach(
+        ({ question }) => (question = reverseWords(question))
```
Comment (SBrandon0432) [inline src/Components/DisplayReveresQuestions/utils.ts:23]: won't set the question as the reversed question.

Comment (hamstra) [reply src/Components/DisplayReveresQuestions/utils.ts:23]: You want map, not for each

Comment (hamstra) [reply src/Components/DisplayReveresQuestions/utils.ts:23]: It will return a new array that you set to a new variable

**Review by SBrandon0432 — COMMENTED** [review]
complete failure on every front,
took way too long for a simple issue
debugging skills were non-present otherwise you would have caught the issue
styling is nonpresent
this should have been 15 min of code and was enormously longer than that, with almost nothing to show for it.

**Pull request closed without merging**